t/admin/plugins.t

```diff
@@ -61,6 +61,16 @@ GET /apisix/admin/plugins/limit-req
 --- request
 GET /apisix/admin/plugins/node-status
 --- response_body
-{"additionalProperties":false,"type":"object"}
+{"properties":{"disable":{"type":"boolean"}},"additionalProperties":false,"type":"object"}
```

Comment:
![image](https://user-images.githubusercontent.com/2106987/93445031-669ced80-f903-11ea-8f87-d386cdddcd12.png)

@membphis Not working currently